docs/data_management/transfer/rclone.md

```diff
@@ -0,0 +1,250 @@
+- Follow the steps in the appropriate section under [Authenticating to Cloud Remotes](#authenticating-to-cloud-remotes), as in the original setup.
+
+## Usage
+
+RClone is a powerful tool with many commands available. We will only cover a small subset of the available commands, as most are beyond typical usage, so please see the [RClone documentation](https://rclone.org/docs/) for more information.
+
+All commands have access to the [global flags](https://rclone.org/flags/). An important global flag is `--dry-run` to show what will happen without actually executing the command, which can be helpful to prevent costly mistakes. [Other Helpful Global Flags](#other-helpful-global-flags) are also available.
+
+The various remotes each have their own individual page with their own specific flags, and are linked in the relevant [Setting up Remotes](#setting-up-remotes) section above.
+
+<!-- markdownlint-disable MD046 -->
+!!! important
+
+    Remote paths are always prefixed by the name of the remote like `cheaha:/path/to/files`. The color character `:` is required for all remote paths. Local paths have no prefix like `/path/to/local/files`. RClone can thus be used between any two machines that are configured where `rclone` is being used, including from the local machine to itself. In the following instructions, replace `<remote:>` by the appropriate remote name from configuration. To access local files, leave `<remote:>` off entirely.
+<!-- markdownlint-enable MD046 -->
+
+<!-- markdownlint-disable MD046 -->
+!!! important
+
+    Remember to use quotes `"` around paths with spaces like `"path\to\a folder with spaces"`
+<!-- markdownlint-enable MD046 -->
+
+### Creating a Directory
+
+To create a directory use `rclone mkdir <remote:><path>`.
+
+Example: `rclone mkdir box:manuscript`.
+
+### Listing Files and Directories
+
+To list files on a machine use `rclone ls <remote:><path>`.
+
+Example `rclone ls box:`.
+
+To list directories on a machine use `rclone lsd <remote:><path>`.
+
+Example: `rclone lsd box:` should show `manuscript`.
+
+### Copying Files
+
+To copy files without changing their name, or to recursively copy directory content, use `rclone copy <source:><path> <destination:><path>`. Note that the directory contents are copied, so when copying a directory, be sure that directory exists on the remote and that you are copying into it.
+
+Example: `rclone copy "C:\users\Name\My Documents" box:manuscript`
+
+To copy a single file and change its name, use `rclone copyto <source:><path/oldname> <destination:><path/newname>`.
+
+Example `rclone copyto "C:\users\Name\My Documents\manuscript.docx" box:manuscript\newest.docx`
+
+### Syncing Between Two Devices
+
+To make a destination directory's contents identical to a source directory, use `rclone sync <source:><path> <destination:><path>`
+
+Example: `rclone sync cheaha:"C:\users\Name\My Documents" box:manuscript`.
+
```

Comment:
add warning here that syncing with delete data from the source if it is missing from the destination. original data could be deleted this way

Reply:
I had meant to do this but it slipped my mind. Thanks for the reminder!